test/LaunchDarkly.OpenFeature.ServerProvider.Tests/ProviderTests.cs

```diff
@@ -41,26 +41,16 @@ public async Task ItHandlesValidInitializationWhenClientIsImmediatelyReady()
         {
             var provider = new Provider(Configuration.Builder("").Offline(true).Build());
 
-            await provider.Initialize(EvaluationContext.Builder().Set("key", "test").Build());
-            Assert.Equal(ProviderStatus.Ready, provider.GetStatus());
-
-            var eventContent = await provider.GetEventChannel().Reader.ReadAsync();
```

Comment:
I will look at adding tests for the behavior with the client. It would let us know about changes in the behavior that could manifest to customers as bugs.